### DISCUSSION TITLE
Σύγχρονη συμμετοχή στην τάξη και ατομικές παρουσιάσεις προόδου

### DISCUSSION THREAD

**Discussion by p15bako** [courses-ionio/hci#1536] [Announcements]
Σχετικά με την τελική βαθμολόγηση αναφέρετε ότι θα πρέπει να έχει συμπληρωθεί η βάση σε κάθε ένα από τα κριτήρια. Ειδικότερα, για το κριτήριο "Σύγχρονη συμμετοχή στην τάξη και ατομικές παρουσιάσεις προόδου", σε προηγούμενη συζήτηση είχατε αναφέρει ότι αρκεί, οι φοιτητές που δεν μπορούν να έχουν φυσική παρουσία στην Κέρκυρα, να παρουσιάσουν την πρόοδο τους 2-3 φορές. Στην περίπτωση που οι παρουσιάσεις πραγματοποιηθούν δια ζώσης, θα είναι αρκετό ώστε να συμπληρωθεί η βάση στο συγκεκριμένο κριτήριο; Ακόμα και αν οι παρουσιάσεις δεν είναι άριστες (10) ;

Επίσης, σχετικά με το πότε θα γίνουν αυτές οι ενδιάμεσες παρουσιάσεις προόδου, θα υπάρξει σχετική ανακοίνωση;

**Comment by epidrome** [comment]
1. ο βαθμός των παρουσιάσεων είναι κάπου 20% και η συμμετοχή στην τάξη 10%, οπότε με δύο πολύ καλές (7+) παρουσιάσεις η βάση είναι εφικτή
2. πολύ σπάνια κάποιος χάνει την βάση από κακή επίδοση, συνήθως αυτό συμβαίνει από επιπολαιότητα απένταντι στις οδηγίες που είναι όλες διαθέσιμες γραπτά
3. οι ατομικές παρουσιάσεις γίνονται με ραντεβού από την 6η εβδομάδα και μετά, οπότε θα υπάρξει και η σχετική ανακοίνωση

μια πολύ καλή ατομική παρουσίαση σχετίζεται κυρίως με την συνέπεια-τεκμηρίωση των άλλων δραστηριοτήτων του μαθήματος, αφού βασικός στόχος της ομάδας είναι να βεβαιωθούμε ότι υπάρχει ευθυγράμμιση με τις ασκήσεις της τελικής γραπτής αναφοράς

**Reply by mariosconsta** [reply]
> 1. ο βαθμός των παρουσιάσεων είναι κάπου 20% και η συμμετοχή στην τάξη 10%, οπότε με δύο πολύ καλές (7+) παρουσιάσεις η βάση είναι εφικτή
> 2. πολύ σπάνια κάποιος χάνει την βάση από κακή επίδοση, συνήθως αυτό συμβαίνει από επιπολαιότητα απένταντι στις οδηγίες που είναι όλες διαθέσιμες γραπτά
> 3. οι ατομικές παρουσιάσεις γίνονται με ραντεβού από την 6η εβδομάδα και μετά, οπότε θα υπάρξει και η σχετική ανακοίνωση
> 
> μια πολύ καλή ατομική παρουσίαση σχετίζεται κυρίως με την συνέπεια-τεκμηρίωση των άλλων δραστηριοτήτων του μαθήματος, αφού βασικός στόχος της ομάδας είναι να βεβαιωθούμε ότι υπάρχει ευθυγράμμιση με τις ασκήσεις της τελικής γραπτής αναφοράς

Μπορεί να γίνει παρουσίαση μέσο zoom;

**Reply by epidrome** [reply]
https://github.com/courses-ionio/hci/discussions/1519
